src/kata6.joinNames.js

```diff
@@ -1,3 +1,10 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  const result = namesObj.map((a) => a.name);
```

Comment:
The solution works perfectly but I would suggest to use little bit more descriptive names.
result could be called namesArray, 
lastArray could be lastName
result1 could be listOfNamesString